Code/src/main/java/nl/utwente/group10/ui/components/blocks/Block.java

```diff
@@ -70,4 +76,13 @@ public final OutputAnchor getOutputAnchor() {
      * @return an expression that evaluates to what this block is.
      */
     public abstract Expr asExpr();
+    
+    /** @return the parent of this Block. */
+    public CustomUIPane getPane() {
+        return parentPane;
+    }
+    
+    /** Invalidates the outputted value and triggers re-evaluation of the value. */
+    public abstract void invalidate();
```

Comment:
In de huidige opzet is deze methode niet logisch op andere blokken dan DisplayBlock.